src/propodesolvers.jl

```diff
@@ -0,0 +1,34 @@
+abstract QuODESolvers <: QuPropagatorMethod
+
+const type_to_method = @compat Dict{Any, Any}(:QuODE45 => ODE.ode45, :QuODE78 => ODE.ode78, :QuODE23s => ODE.ode23s)
+
+for op in keys(type_to_method)
+    @eval  begin
+        @doc """
+        ODE Methods types
```

Comment:
I have posted this on the gitter room of Docile, I guess the way out may be is to create a custom macro as in the issue https://github.com/JuliaQuantum/QuDynamics.jl/issues/5. Other than this issue, is everything fine on this ?